changes: various 8.5.0 bits and bobs (#849)
cylc/cylc-doc: src/user-guide/interventions/index.rst

         $ cylc trigger <workflow>//<task>

.. _interventions.re-run-multiple-tasks:

Re-Run Multiple Tasks
---------------------

:Example:
   I need to re-run a few tasks in order.

:Solution:
   Determine the tasks you want to re-run and trigger all of them at the
   same time.

   We often want to
   :ref:`edit the tasks configuration <interventions.edit-a-tasks-configuration>`
   before doing this.

.. tab-set::

   .. tab-item:: GUI
      :sync: gui

>       .. image:: ./../../reference/changes/group-trigger.gif

Reviewer: Supports my contention that this should be in `src/img`

         $ cylc trigger --flow=new <workflow>//<cycle>/<task>

.. warning::

>    Be aware that tasks which have previously run, will not have run in the

Reviewer 2: ```suggestion
   Be aware that tasks which previously ran will not run again in the
   new flow. As a result, you may want to manually set task outputs to allow
```

>    Be aware that tasks which have previously run, will not have run in the

Reviewer: Warning ~= be aware .. repetitious?
```suggestion
   Tasks which have run before your trigger command will not have run in the
```
